config.yml

```diff
@@ -1,8 +1,8 @@
 settings:
-  accountName: "MyAccountName"
-  deploymentUid: "ss881269"
-  sitecorePrefix: "demo"
+  accountName: "HorizontalIntegration"
+  deploymentUid: "ss300127"
+  sitecorePrefix: "sitecore"
   pathToWWWRoot: "C:\\inetpub\\wwwroot"
   solrUsername: ""
   solrPassword: ""
-  sitecoreVersion : "9.2.0" #Supported versions - 9.0.2, 9.1.1, and 9.2.0
\ No newline at end of file
+  sitecoreVersion : "9.3.0" #Supported versions - 9.0.2, 9.1.1, and 9.2.0
```

Comment:
Should update the example to have 9.3 in there

Reply:
Sure